src/main/java/org/eclipse/tractusx/ssi/examples/VP.java

```diff
@@ -1,6 +1,5 @@
-/*
- * ******************************************************************************
- * Copyright (c) 2021,2023 Contributors to the Eclipse Foundation
+/********************************************************************************
```

Comment:
Seems like the automatic formatting of the copyright header didn't work here. In new files it seems to be fine. I'm referring to the fact that the header starts and ends with `/*\n`  unlike here.